components/log-viewer-webui/server/src/S3Manager.js

```diff
@@ -55,4 +69,8 @@ class S3Manager {
     }
 }
 
-export default S3Manager;
+export default fastifyPlugin(async (app, region) => {
```

Comment:
`fastifyPlugin` requires the second argument in the callback to be an `options` object: See https://fastify.dev/docs/latest/Reference/Plugins/#plugin-options 

because Fastify specific options like `logLevel`, `logSerializers`, `prefix` are expected to show up in the options object if they exist.

```suggestion
export default fastifyPlugin(async (app, options) => {
```